cosmoz-tab.html

```diff
@@ -125,8 +123,15 @@
 				@apply --layout-wrap;
 			}
 
-			:host([has-cards][accordion])  #content {
-				display: block;
+			:host(:not([has-cards])[accordion]) #content {
+				transition-duration: var(--cosmoz-tab-collapse-transition-duration, 300ms);
```

Comment:
`--cosmoz-tab-collapse-transition-duration` should be `--cosmoz-tab-transition-duration` similar to the variable for tab-card